client/src/components/EventForm/index.jsx

```diff
@@ -115,46 +156,51 @@ class AddEvent extends Component {
             <Input
               s={12}
               id="name"
```

Comment:
I think better change id, It does not describe who have this name.

Reply:
It will crash all logic because we have 'name' field in db. input id -> state -> request data -> colander validation -> pyramid view -> db. On all these steps we have field 'name', so change it on one of these steps are not very clever.

Reply:
So why I haven't same problems with my id?

Reply:
If you talk about recover password task because:
first, fields for recover password don't have direct contact with db, but my fields have.
second, we had a conflict with a password for log-in at navbar, so we had to change it